lib/mayaHydra/hydraExtensions/sceneIndex/mhMayaUsdProxyShapeSceneIndex.cpp

```diff
@@ -331,18 +333,19 @@ Fvp::PrimSelections MayaUsdProxyShapeSceneIndex::UfePathToPrimSelections(
     const Ufe::Path& appPath
 ) const
 {
-    // We only handle USD objects, so if the UFE path is not a USD object,
```

Comment:
Remove single UFE run-time restriction, path mapping should only deal with paths.